src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -0,0 +1,105 @@
+(ns quo2.components.wallet.account-card.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [quo2.components.wallet.account-card.style :as style]
+            [quo2.components.buttons.button.view :as button]
+            [utils.i18n :as i18n]
+            [quo2.components.markdown.text :as text]
+            [quo2.theme :as theme]))
+
+(defn loading-view
+  [{:keys [customization-color type theme metrics?]}]
+  (let [watch-only? (= :watch-only type)
+        empty?      (= :empty type)]
+    [rn/view (style/card customization-color watch-only? metrics? theme)
+     [rn/view {:style {:flex-direction :row :align-items :center}}
+      [rn/view {:margin-right 8 :margin-top 2 :style (style/loader-view 16 16 watch-only? theme)}]
+      [rn/view {:style style/watch-only-container}
+       [rn/view {:style (style/loader-view 57 8 watch-only? theme)}]
+       (when watch-only? [icon/icon :reveal {:color colors/neutral-50 :size 12}])]]
+     [rn/view {:margin-top 13 :style (style/loader-view (if empty? 56 80) 16 watch-only? theme)}]
+     (when metrics?
+       [rn/view
+        {:margin-top 10
+         :style      (style/loader-view (if empty? 37 96) 8 watch-only? theme)}])]))
+
+(defn user-account
+  [{:keys [state name balance percentage-value loading? amount customization-color type emoji metrics?
+           theme]}]
+  (let [watch-only?        (= :watch-only type)
+        empty?             (= :empty type)
+        account-amount     (if (= :empty state) "€0.00" amount)
```

Comment:
I think all these values should be translated @FFFra. Euros don't make sense for most users in the world and `Account 1` is also in English.

Numbers should be formatted according to their localization. Take a look at the `utils.i18n/format-currency` function, it might be just what you need.

Reply:
I agree, but as far as I know,  fiat currencies and symbol handlers are not defined in this case. I will double-check with the team if I can change that now. Nice catch! 

@J-Son89, what do you think?

Reply:
Let's discuss this with the team first :)

Reply:
I'm checking the codebase to confirm the best approach and will back to you guys

Reply:
Nice, yeah, let's always align with designers if the examples are not in Figma. Thanks @FFFra